query/src/sql/planner/mod.rs

```diff
@@ -13,6 +13,7 @@
 // limitations under the License.
 
 use std::sync::Arc;
+use std::sync::RwLock;
```

Comment:
It's better to use the common/base/core/infallible rwlock https://github.com/datafuselabs/databend/blob/main/common/base/src/infallible/rwlock.rs , so we no need do any unwrap.

Reply:
That's good. 

By the way, I'd like to know if it is the canonical `Mutex` of databend?

Reply:
Infallible rwlock is a simple wrapper around the lock() function of a std::sync::RwLock The only difference is that you don’t need to call unwrap() on it.

Infallible Mutex https://github.com/datafuselabs/databend/blob/main/common/base/src/infallible/mutex.rs

Reply:
Fixed.